pkgs/by-name/va/vacuum-tube/package.nix

```
@@ -0,0 +1,72 @@
+{
+  buildNpmPackage,
+  fetchFromGitHub,
+  lib,
+  electron,
+  makeWrapper,
+  writableTmpDirAsHomeHook,
+}:
+
+buildNpmPackage rec {
+  name = "vacuum-tube";
```

```suggestion
  pname = "vacuum-tube";
```
otherwise we overwrite name and version is lost